src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -429,5 +429,89 @@ public static String getCopyrightAcceptUrl(String copyrightAlert,
 		}
 		return contentUrl;
 	}
+	
+	/**
+	 * change folder path based on updated folder title
+	 * @param folderNameUpdates
+	 * @param title
+	 * @param folderName
+	 * @return
+	 */
+	public static HashMap<String, String> updateFolderName(HashMap<String, String> folderNameUpdates,
```

Comment:
Is it possible if there's was a directory tree like test/test and only one of the folder names was renamed, that this method will inadvertently rename both instances?